src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -88,7 +91,7 @@ class CrashHeader extends React.Component {
     const {title, beforeTitle, hideGuide, stackView, stackType, newestFirst} = this.props;
 
     let titleNode = (
-      <h3 className="pull-left">
+      <h3 className="pull-left" style={{marginBottom: space(0)}}>
```

Comment:
I would like to see us not use inline styles, but changing this to use styled-components is a bit of out of scope for this PR so 👍 

I think we can use 0 here - it seems safe to always assume that `space(0) === 0` and that the grid sizing for `0` would never be non-zero